Bottom: fca90867bde9e3793efd01e53e619e93e15c03c3
Top:    658798d33cb5f6284644dec345a9d5e3352594f5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-16 16:08:05 +0100

Refresh of use-trackball-to-allow-rotatio

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index a8404e9..20c6533 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -801,7 +801,10 @@ static GLfloat view_matrix[4][4] = {{1.0, 0.0, 0.0, 0.0},
 void
 ghid_port_rotate (void *ball, float *quarternion, gpointer userdata)
 {
+#ifdef DEBUG_ROTATE
   int row, column;
+#endif
+
   build_rotmatrix (view_matrix, quarternion);
 
 #ifdef DEBUG_ROTATE
